### PR TITLE
{lib}[intel/2017b] PIMS v0.4.1, nd2reader v3.0.6 w/ Python 2.7.14

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#5543~~ (scikit-image + matplotlib), ~~#5544~~ (FFmpeg)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2499.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/897c36b26d5aba2c6f86bd85618a1982 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2088.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/603fcb4c31c35d4786ba40746b66e2c3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2423.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4adf39c991eb3e4a503ad925258bb79b for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/6b3e3dea40a76e824431564b2a230526 for a full test report.

File: easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+full_sanity_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+moduleclass = 'lib'
```
Comment (verdurin) [inline easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb]: @boegel - did wonder whether these should be `bio`, but it does indeed look as though they're generic.

File: easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+easyblock = 'Bundle'
+
+name = 'nd2reader'
+version = '3.0.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://soft-matter.github.io/pims'
+description = "PIMS is a lazy-loading interface to sequential data with numpy-like slicing."
```
Comment (verdurin) [inline easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb]: @boegel - this homepage and description don't look right. Shouldn't they be taken from https://github.com/soft-matter/pims_nd2?

File: easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb]: Not `bio` imho, it's a generic library for reading image/video formats, afaik.

File: easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb]: Ugh, copy-paste mistake, will fix, thanks.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**